### PR TITLE
fixed toast for multiple components selection

### PR DESCRIPTION
**Pull request by ChaitanyaKharya2003**
Fixed Issue #7226

### CODE REVIEW

**Comment by tejakummarikuntla** [comment]
Hey @ChaitanyaKharya2003 , thanks for the PR. Please sign the CLA to progress.

File: frontend/src/_helpers/appUtils.js [hunk 1]
```diff
@@ -1349,7 +1349,11 @@ export const cloneComponents = (_ref, updateAppDefinition, isCloning = true, isC
     updateAppDefinition(newDefinition);
   } else {
     navigator.clipboard.writeText(JSON.stringify(newComponentObj));
-    toast.success('Component copied succesfully');
+    if (newComponentObj.newComponents.length > 1) {
```
Comment (Ckbhatia) [inline frontend/src/_helpers/appUtils.js:1352]:
```suggestion
    const successMessage = newComponentObj.newComponents.length > 1 ? "Components copied successfully" : "Component copied successfully";
    toast.success(successMessage);
```

File: frontend/src/_helpers/appUtils.js [hunk 2]
```diff
@@ -1349,7 +1349,8 @@ export const cloneComponents = (_ref, updateAppDefinition, isCloning = true, isC
     updateAppDefinition(newDefinition);
   } else {
     navigator.clipboard.writeText(JSON.stringify(newComponentObj));
-    if (newComponentObj.newComponents.length > 1) {
+    const successMessage = newComponentObj.newComponents.length > 1 ? "Components copied successfully" : "Component copied successfully";
+    toast.success(successMessage);
       toast.success("Components copied succesfully");
```
Comment (Ckbhatia) [inline frontend/src/_helpers/appUtils.js:1354]: You can remove this and else statement with code as you don't need anymore

**Review by stepinfwd — CHANGES_REQUESTED** [review]
Remove code from 1354 to 1357 @ChaitanyaKharya2003 we dont need else case and additional toast

**Comment by nakulnagargade** [comment]
> Remove code from 1354 to 1357 @ChaitanyaKharya2003 we dont need else case and additional toast

@ChaitanyaKharya2003 Please resolve this active comment. Else looks fine.

**Comment by tejakummarikuntla** [comment]
Hey @ChaitanyaKharya2003, could you please address the reviews? We would not be able to progress it to merge without the suggested changes.

**Review by kavinvenkatachalam — APPROVED** [review]

**Review by stepinfwd — APPROVED** [review]

**Pull request merged**